transformation-engine/latest/install/msi.md

```diff
@@ -31,6 +31,8 @@ In previous versions the installation files were contained within a `.zip` file.
 
     > **Important:** If you do not intend to use the DTE T-Engine Docker image, you must select this option for DTE to work correctly.
 
+    > **Note:** Currently, only Alfresco Content Services 7.x is compatible with the T-Engine approach.
```

Comment:
I think we need to rewrite this sentence here. To clarify that:
You can use ACS 6.x with the T-Engine approach and with the old approach by installing the AMPs on ACS.

For ACS 7.x you can only use the T-Engine approach for now. AMP installation is not possible.